src/Uplink/Auth/Admin/Connect_Controller.php

```diff
@@ -179,5 +179,14 @@ public function maybe_store_token_data(): void {
 		 * @param \StellarWP\Uplink\Resources\Resource $plugin The plugin that was connected.
 		 */
 		do_action( 'stellarwp/uplink/' . Config::get_hook_prefix() . '/' . $slug . '/connected', $plugin );
+
+		/**
+		 * Fires after a plugin has been connected.
+		 *
+		 * @since 2.2.2
+		 *
+		 * @param \StellarWP\Uplink\Resources\Resource $plugin The plugin that was connected.
+		 */
+		do_action( 'stellarwp/uplink/' . Config::get_hook_prefix() . '/connected', $plugin );
```

Comment:
```suggestion
		do_action( 'stellarwp/uplink/' . Config::get_hook_prefix() . '/connected', $plugin, $slug );
```

Just a suggestion, would passing `$slug` be helpful?

Reply:
Nope we pass $plugin - slug is just `$plugin->get_slug()` ;)